action.yml

```diff
@@ -13,7 +13,8 @@ inputs:
     default: 'Debug'
 runs:
   using: 'docker'
-  image: 'docker://ghcr.io/apollo-fire/ccs-build:v0.0.3'
+  image: Dockerfile
+  # image: 'docker://ghcr.io/apollo-fire/ccs-build:v0.0.4'
```

Comment:
if this is not used anymore can it be deleted?

Reply:
The raw Dockerfile is used to test things before merging, the PR needed changing to the image to allow it to be used once released.

Good catch!